src/components/dateTimePicker/index.js

```diff
@@ -249,6 +249,7 @@ class DateTimePicker extends Component {
           is24Hour={is24Hour}
           minuteInterval={minuteInterval}
           timeZoneOffsetInMinutes={timeZoneOffsetInMinutes}
+          display={Platform.OS === "ios" ? "inline" : "default"}
```

Comment:
You don't have Platform imported. 
Anyway, better use our Constants 
```suggestion
          display={Constants.isIOS ? "inline" : "default"}
```